### PR TITLE
feat:add database

### PR DESCRIPTION
**Pull request by HarryPotter-4000**
I added 4 different quizzes, 5 and 10 questions each. JS with 2 possible answers. 
I'm not sure about the structure, so, please, check it.

### CODE REVIEW

File: src/data/fakeData.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    id: 1,
+    question: "What is a hook?",
+    answers: [
+      "a JavaScript library for building user interfaces",
+      "a javascript function, allows you to create/access state and life cycles",
+      "one of the core building blocks of React",
+      "the standard routing library for React",
+    ],
+    isCorrect: answers[1],
+    explanation: string, // (if we wanna display it)
```
Comment (lazzzaroni) [inline src/data/fakeData.js]: Change value of this field to some placeholder (like `"Explanation will be here"`) or add a real explanation on every field in the dock, please. One or two short sentences should be enough.
It seems that we will need this data in the future 🙂

File: src/data/fakeData.js [hunk 2]
```diff
@@ -0,0 +1,283 @@
+const reactRules = [
+  {
+    id: 1,
+    question: "What is a hook?",
```
Comment (lazzzaroni) [inline src/data/fakeData.js]: Capitalise `"Hook"` (just to keep it consistent across all questions)

File: src/data/fakeData.js [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+    answers: [
+      "True",
+      "False"
+    ],
+    isCorrect: answers[0],
+    explanation: string, // (if we wanna display it)
+  },
+  {
+    id: 5,
+    question: "What will be the result let a = (1,5 - 1) * 2 ?",
```
Comment (lazzzaroni) [inline src/data/fakeData.js]: Format this and `id: 9` question like the last one, e.g.
`"What will be the result: let counter = 0; alert( counter++ ); ?"`

I'm feeling myself like a nerd 🫠

File: src/data/fakeData.js [hunk 4]
```diff
@@ -0,0 +1,283 @@
+const reactRules = [
+  {
+    id: 1,
+    question: "What is a hook?",
+    answers: [
+      "a JavaScript library for building user interfaces",
+      "a javascript function, allows you to create/access state and life cycles",
```
Comment (lazzzaroni) [inline src/data/fakeData.js]: I suggest this approach: 
- if it is a sentence => capitalise the first word
- if it is a tag/attributes/code => leave it as it is
- if it is a boolean => remove capitalisation

File: src/data/fakeData.js [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+  {
+    id: 3,
+    question: "What will be the result var foo = 10 + "20" ?",
+    answers: ["30", "'1020'"],
+    isCorrect: answers[1],
+    explanation: string, // (if we wanna display it)
+  },
+  {
+    id: 4,
+    question: "'Use strict mode' helps in writing cleaner and safer code?",
```
Comment (lazzzaroni) [inline src/data/fakeData.js]: `'use strict'` mode

**Comment by lazzzaroni** [comment]
Great job, btw!

File: src/data/fakeData.js [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+  {
+    id: 1,
+    question: "What is a hook?",
+    answers: [
+      "a JavaScript library for building user interfaces",
+      "a javascript function, allows you to create/access state and life cycles",
+      "one of the core building blocks of React",
+      "the standard routing library for React",
+    ],
+    isCorrect: answers[1],
```
Comment (LizavetaY) [inline src/data/fakeData.js]: `isCorrect` means a boolean value. Change it to `correctIndex` in your example

File: src/data/fakeData.js (hunk 1 above)
Comment (HarryPotter-4000) [reply src/data/fakeData.js]: done

**Comment by HarryPotter-4000** [comment]
`answers: [
  { id: 'ans-1', data: 'Answer bla-bla-bla' },
  { id: 'ans-2', data: 'Answer blo-blo-blo' },
  ...
]
и, соответственно:
correctAnswerId: 'ans-2'`

**Review by LizavetaY — APPROVED** [review]

**Review by lazzzaroni — APPROVED** [review]

**Pull request merged**